coding_style_and_standards.md

```diff
@@ -100,18 +99,23 @@ documentation server which regenerates the docs on each request.
     SHA references to other commits, and even raw data to make the purpose
     of the commit clearer (e.g. "Was broken by commit 0f3a459b").
   * A blank line if you've written a body.
-  * References to any Bugzilla tickets or Github Issues, one per line if
+  * References to any GitHub Issues or other ticketing systems, one per line if
```

Comment:
The whole www thing seems overly complex in this context.  I wonder if we could have like a Tips and Tricks page and then link to that for that purpose.

Reply:
Ok, how about ignoring the workaround and make people aware they will be backlinked from any github issue/PR they reference:

```
Note: ManageIQ repositories are public. If you reference a public GitHub issue or pull request, it will be referenced back to your issue, pull request or commit.
```